aspects/crossplane-promise/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"log"
+
+	"github.com/syntasso/kratix-cli/aspects/helm-promise/lib"
+	"github.com/syntasso/kratix-cli/cmd"
+)
+
+func main() {
+	group := lib.GetEnvOrDie(cmd.XRD_GROUP_ENV_VAR)
+	version := lib.GetEnvOrDie("XRD_VERSION")
+	kind := lib.GetEnvOrDie("XRD_KIND")
```

Comment:
fix: pick one 👀 (either the constant or the string value)

Reply:
this is what happens when you respond to slack mid refactor